src-web/components/common/TemplateEditor/TemplateEditor.js

```diff
@@ -42,7 +42,7 @@ const tempCookie = 'template-editor-open-cookie'
 const diagramIconsInfoStr = '#diagramIcons_info'
 // Regex to test valid policy name format
 // (it's 63 characters for `${policyNS}.${policyName}`)
-const policyNameRegex = RegExp(/^[a-z0-9][a-z0-9-.]{0,63}[a-z0-9]$/)
+const policyNameRegex = RegExp(/^[a-z0-9][a-z0-9-.]{0,61}[a-z0-9]$/)
```

Comment:
Hi @ChunxiAlexLuo , out of curiosity, why did you decrease the value by 2?

Reply:
because this regex is testing 63

you can try it here
https://regex101.com/
`default.this-is-55-characters-long-policy-name-xxxxxxxxxxxxxxxx` is total length 63
<img width="854" alt="Screen Shot 2021-03-17 at 9 01 30 AM" src="https://user-images.githubusercontent.com/55596988/111471582-6c7f5a00-86ff-11eb-9687-da1c64127025.png">

`default.this-is-56-characters-long-policy-name-xxxxxxxxxxxxxxxxy` is total length 64
<img width="854" alt="Screen Shot 2021-03-17 at 9 01 42 AM" src="https://user-images.githubusercontent.com/55596988/111471596-70ab7780-86ff-11eb-863b-3caa73318416.png">

Reply:
Also for a total length of 62
<img width="854" alt="Screen Shot 2021-03-17 at 9 02 40 AM" src="https://user-images.githubusercontent.com/55596988/111471726-95075400-86ff-11eb-9651-10e23cea559b.png">

Reply:
The error message speaks about combined length of policy name and namespace and does not mention the dot. My understanding was that the actual limit is 64 characters and 63 is mentioned because the extra dot ('.') used to connect the two.

Reply:
No, the limit is 63.

Reply:
I see. Thing thing is that the error message doesn't say that which is slightly confusing. Maybe it should say that
```
the combined length of namespace and policy name should not exceed than 62 characters
```
or something like
```
the combined length of "Namespace.PolicyName" should not exceed than 63 characters
```

Reply:
sounds good to me. This is what we have in doc
<img width="734" alt="image" src="https://user-images.githubusercontent.com/16092291/111479402-52497a00-8707-11eb-8a1d-973578898027.png">

Reply:
pushed a commit to change it to `should not exceed 62 characters`

Reply:
To be precise, the message above says it should be less than (not equal to) 63 chars including the dot. Maybe documentation needs fixing too. :-) Anyway, given the documentation mentions namespaceName.policyName already, I think the error message should mention the pattern too to be consistent.

Reply:
`the combined length of namespace and policy name (namespaceName.policyName) should not exceed 63 characters`
Is it better?

Reply:
Also, additional updates are necessary in https://github.com/open-cluster-management/grc-ui/blob/main/tests/cypress/tests/create_policy_field_validation.spec.js on top the updated error message:

Depending on the actual limit (62 or 63) we need to 
- update lines 15-18 since the current policy name used won't meet the new limit, the 'o' prefix in the policy name should be removed, policy name updated to meet the new limit and comments updated accordingly
- remove line 21 and update policy name on line 22 to meet the new limit (if it won't already)

Reply:
And one last question related to https://bugzilla.redhat.com/show_bug.cgi?id=1939409#c1 and the Kubernetes object name limit. When I create a policy `foo` also the PlacementBinding `binding-foo` and PlacementRule `placement-foo`gets created. Does the Kubernetes limit apply also to these two objects? Because if it does, it would imply additional limit on the policy name length, since `placement-` prefix has length 10, most probably exceeding the length of the used namespace.

Reply:
> `the combined length of namespace and policy name (namespaceName.policyName) should not exceed 63 characters`
> Is it better?

Good for me, assuming that the documentation is wrong stating "less than 63" and would be fixed too.

Reply:
I have asked @dockerymick to fix the doc. Btw, good catch on placement rule, I am not sure why it would go through. Looks like there is no restriction on it. I will look into it.

Reply:
OK. Thank you. I have no further requests except the (minor) comment addition.